alchemist/alchemist-incarnation-biochemistry/build.gradle

```diff
@@ -6,6 +6,7 @@ sourceSets {
 
 dependencies {
     compile (
+        project(':alchemist-loading'),
```

Comment:
then it is wrong. It does not belong to the compile configuration, but to the `testCompile` configuration.
```groovy
testCompile ':alchemist-loading'
```